pkg/util/pki/csr.go

```diff
@@ -179,13 +112,6 @@ func KeyUsagesForCertificateOrCertificateRequest(usages []v1.KeyUsage, isCA bool
 	return
 }
 
-func BuildCertManagerKeyUsages(ku x509.KeyUsage, eku []x509.ExtKeyUsage) []v1.KeyUsage {
```

Comment:
This function was not yet deprecated, but is likely not used outside of cert-manager and can easily be replaced.